src/modules/75-cd/components/PipelineSteps/Common/Terraform/Editview/TerraformEditView.tsx

```diff
@@ -306,7 +306,7 @@ export default function TerraformEditView(
                                 return (
                                   <TFMonaco
                                     name="spec.configuration.spec.backendConfig.spec.content"
-                                    formik={formik}
+                                    formik={formik as any}
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
                                    formik={formik as unknown}
```